\hypertarget{catch__config__uncaught__exceptions_8hpp}{}\doxysection{catch\+\_\+config\+\_\+uncaught\+\_\+exceptions.\+hpp File Reference}
\label{catch__config__uncaught__exceptions_8hpp}\index{catch\_config\_uncaught\_exceptions.hpp@{catch\_config\_uncaught\_exceptions.hpp}}
{\ttfamily \#include $<$exception$>$}\newline


\doxysubsection{Detailed Description}
\doxylink{struct_wrapper}{Wrapper} for UNCAUGHT\+\_\+\+EXCEPTIONS configuration option

For some functionality, Catch2 requires to know whether there is an active exception. Because {\ttfamily std\+::uncaught\+\_\+exception} is deprecated in C++17, we want to use {\ttfamily std\+::uncaught\+\_\+exceptions} if possible. 